src/main/resources/assets/securitycraft/lang/fr_fr.json

```diff
@@ -0,0 +1,767 @@
+	"gui.securitycraft:scManual.viewActivatedBloc": "Affichage activé: ce bloc peut être activé en le survolant avec votre curseur lorsque vous êtes à portée.",
+	"gui.securitycraft:scManual.explosiveBloc": "Explosif: ce bloc peut être déclenché à distance, et peut être activé / désactivé, à l’aide de l’outil d’accès aux mines à distance.",
+	"gui.securitycraft:scManual.customizableBloc": "Personnalisable: vous pouvez utiliser le Modificateur de Bloc universel et les différents modules ajoutés par SecurityCraft pour personnaliser les fonctions de ce bloc.",
+	"gui.securitycraft:scManual.options": "Options disponibles:",
+	"gui.securitycraft:scManual.modules": "Modules disponibles:",
+	"gui.securitycraft:scManual.recipe.keypad": "Placez un cadre et faites un clic droit sur le clavier",
+	"gui.securitycraft:scManual.recipe.keypad_chest": "Placez un Coffre et faites un clic droit sur le clavier.",
+	"gui.securitycraft:scManual.recipe.keypad_furnace": "Placez un Four et faites un clic droit sur le clavier.",
+	"gui.securitycraft:scManual.recipe.reinforced": "Tout bloc blindé peut être fabriqué comme suit, sauf indication contraire dans ce livre. Placez l’équivalent du bloc standard (par exemple Pierre blindée -> Pierre) et cassez-le avec un Blindeur de blocs universel, ou cliquez avec le bouton droit sur le Blindeur de blocs universel, mettez-le dans la fente et fermez l’interface graphique.",
+	"gui.securitycraft:scManual.recipe.reinforced_iron_trapdoor": "Tout bloc blindé peut être fabriqué comme suit, sauf indication contraire dans ce livre. Placez l’équivalent du bloc standard (par exemple Pierre blindée -> Pierre) et cassez-le avec un Blindeur de blocs universel, ou cliquez avec le bouton droit sur le Blindeur de blocs universel, mettez-le dans la fente et fermez l’interface graphique.",
+	"gui.securitycraft:scManual.recipe.adminTool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration.",
+	"gui.securitycraft:scManual.recipe.admin_tool": "L’outil d’administration est uniquement disponible via l’inventaire, s’il est activé via le fichier de configuration..",
+	"gui.securitycraft:scManual.recipe.reinforced_crystal_quartz": "Fabriqué à partir de Bloc de Cristal de Quartz ordinaire dans un Blindeur de blocs universel.",
+	"gui.securitycraft:scManual.recipe.sc_manual": "Le manuel SecurityCraft est uniquement disponible via l’inventaire ou en tapant \"/sc help\" (sans apostrophes).",
+	"gui.securitycraft:scManual.reinforced": "Blocs blindés",
+	"gui.securitycraft:scManual.designedBy": "Conçu par: %s",
+	"gui.securitycraft:ims.hostileAndPlayers": "Mobs hostiles & joueurs",
+	"gui.securitycraft:ims.hostile": "Mobs hostiles",
+	"gui.securitycraft:ims.target": "Cible:",
+	"gui.securitycraft:invScan.emitredstone": "Emettre redstone",
```

Comment:
You accidentally changed the casing of the last part of the language keys here. It should be `emitRedstone`, not `emitredstone`.